src/callbacks_step/analysis_surface_integral_2d.jl

```diff
@@ -48,31 +48,31 @@ struct AnalysisSurfaceIntegral{Variable}
     end
 end
 
-struct ForceState{RealT <: Real}
+struct FlowStateDirectional{RealT <: Real}
```

Comment:
Why the name change, or at least why the "Directional" suffix? Are their other states that might be considered? Maybe "FreestreamFlowState" would be more appropriate?

Also, what does `psi` stand for?

Reply:
Regarding the name change, I felt `FlowState` would be more descriptive.

`Directional` is since this flow state carries the vector of the free-stream velocity `uinf`.

The non-directional version of this has only the remaining three quantities.